### ISSUE TITLE
Can't Fully  Hide Touch Screen Controls

### ISSUE THREAD

**Issue by doctorwotm8** [Waterdish/Shipwright-Android#61]
The touch screen control overlay covers the screen, and it can be hidden by tapping the eyeball in the lower right corner. However, there is no way to make the eyeball go away.

**Comment by VorgonBattlecruiser** [comment]
yeah, not great for OLED screens, is there no way to hide the eye?

**Comment by NeilGraham** [comment]
I agree, I would like to be able to hide the eye within Settings > Controls
